pages/project/read/views/index.jsx

```diff
@@ -20,14 +20,13 @@ const getVersions = model => {
   return versions;
 };
 
-const App = ({ model, establishment, canUpdate }) => {
+const App = ({ model, establishment, canUpdate, isGrantedVersion }) => {
   const openTask = model.openTasks.find(task => task.status !== 'returned-to-applicant');
-
   const canAmend = canUpdate && model.status === 'active' && !openTask;
 
   return (
     <Fragment>
-      <LicenceStatusBanner licence={model} licenceType="ppl" />
+      <LicenceStatusBanner licence={model} licenceType="ppl" isGrantedVersion={isGrantedVersion} />
```

Comment:
Having now gone to look at https://github.com/UKHomeOffice/asl-components/pull/48 I'm struggling to see where the `isGrantedVersion` prop is actually used in the component.